clang/lib/Parse/ParseDeclCXX.cpp

```diff
@@ -81,7 +81,7 @@ Parser::DeclGroupPtrTy Parser::ParseNamespace(DeclaratorContext Context,
 
   ParsedAttributes attrs(AttrFactory);
 
-  auto ReadAttributes = [&] {
+  auto ReadAttributes = [&](bool TrailingAttrs) {
```

Comment:
Pre-existing, but why is this not using MaybeParseAttributes ? @AaronBallman

Reply:
@erichkeane @AaronBallman could you please review these changes? thanks